bin/main.cpp

```diff
@@ -13,17 +31,28 @@ int main(int argc, char** argv)
     while(App.pollEvent(Event))
     {
       // Exit
-      if(Event.type == sf::Event::Closed)
-        App.close();
+      if(Event.type == sf::Event::Closed) {
+          App.close();
+      }
+      else if (Event.type == sf::Event::LostFocus) {
+          focused = false;
+      }
+      else if (Event.type == sf::Event::GainedFocus) {
+          focused = true;
+      }
+    
+      
     }
-
-    // clear screen and fill with blue
-    App.clear(sf::Color::Blue);
-
-    // display
-    App.display();
+    if (focused) {
+        float deltaMs = clock.restart().asSeconds();
```

Comment:
Since the variable name is deltaMs, shouldn't this be asMilliseconds instead of asSeconds?